general/releases.md

```diff
@@ -75,17 +75,23 @@ The most recent [long-term support release (LTS)](https://en.wikipedia.org/wiki/
 
 These are the target dates for releases. These dates may vary slightly due to unforeseen circumstances.
 
-| Release type | Frequency | Months |
+| Release type | Frequency | Months | Timing |
 | --- | --- | --- |
-| [Major](./development/process.md#major-release-cycles) (eg. 3.x) | 6 monthly | Second Monday of May and November |
-| [Minor](./development/process.md#stable-maintenance-cycles) (Point) (eg. 3.x.y) | 2 monthly | Second Monday of July, September, November, March and May. Third Monday of January. |
+| [Major](./development/process.md#major-release-cycles) (eg. 3.x) | 6 monthly | April, and October | Weeks 17, and 43 of the year |
+| [Minor](./development/process.md#stable-maintenance-cycles) (Point) (eg. 3.x.y) | 2 monthly | February, April, June, August, October December | Usually every 8 weeks |
```

Comment:
Might be better to use "Usually every 2 months" instead of "Usually every 8 weeks". Probably good also to indicate the timing of the release is on the 2nd Monday of the release month.